mlrun/api/schemas/tag.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2018 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import typing
+
+import pydantic
+
+from .artifact import ArtifactObject
+
+
+class TagObject(pydantic.BaseModel):
+    """Tag object"""
+
+    kind: str
+    # TODO: Add more types to the list for new supported tagged objects
+    identifiers: typing.List[typing.Union[ArtifactObject]]
+
+
+class TagsObjects(pydantic.BaseModel):
```

Comment:
No need for this class for now - if in the future we would want to support operations on multiple types of objects in the same REST call we can add it, but for now it seems to be redundant. TBH, I don't think we'll support it also in the future...